daemon/cmd/daemon.go

```diff
@@ -1112,6 +1104,23 @@ func NewDaemon(ctx context.Context, cancel context.CancelFunc, epMgr *endpointma
 	d.ipcache.InitIPIdentityWatcher()
 	identitymanager.Subscribe(d.policy)
 
+	// Start listening to changed devices if requested.
+	if option.Config.EnableRuntimeDeviceDetection {
+		if linuxdatapath.AreDevicesRequired() {
+			devicesChan, err := d.deviceManager.Listen(ctx)
+			if err != nil {
+				log.WithError(err).Warn("Runtime device detection failed to start")
+			}
+			go func() {
+				for devices := range devicesChan {
+					d.ReloadOnDeviceChange(devices)
+				}
+			}()
+		} else {
+			log.Warn("Runtime device detection requested, but no feature requires it. Disabling detection.")
```

Comment:
Why `Warn`? This looks like it should be `Info`. If the user configured the particular set of options, then it's nothing to warn about, this is an OK state for the daemon to be in. We let the user know in case it surprises them, but it should not cause any unexpected consequences.

Reply:
Changed to info.